src/Snackbar/Snackbar.js

```diff
@@ -197,6 +202,8 @@ class Snackbar extends Component {
     }, 400);
   }
 
+  contentId = UniqueId('snackbar');
```

Comment:
Great catch! Am I right in thinking that would be because the ID generator relies on a single instance of itself to assure uniqueness and a server rendered page would have an instance on server and on client?

Perhaps exposing `contentId` as a prop on the [Snackbar] component and passing it down to [SnackbarBody] could be a cleaner solution?